doc/source/generate_module_rst.py

```diff
@@ -64,6 +64,7 @@
 SPHINX_INCLUDE_MODULE_STRING = """
 {name}
 {level}
+.. automodule:: {name}
```

Comment:
This is what was missing to pull module-level docstring in at all.

Reply:
So it wasn't the auto generation part that caused the missing docstrings, right?

Reply:
Yes, correct!

Reply:
great spot gang